test/libyul/yulSyntaxTests/smok.yul

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Weird filename...

Reply:
It also shares the language roots (`smjúga/smaug`) with Smaug from lotr